src/DGtal/shapes/doc/moduleSurfaceMesh.dox

```diff
@@ -235,5 +235,76 @@ corresponding to relative distances 0.25, 0.5 and 0.75.
 
 @image html spot-bft-isolines.png "SurfaceMesh faces colored according to distance to bluest vertex and three isodistance lines"
 
+\section Flipping edges
+
+Since 1.4, you may modify a SurfaceMesh through flips. Only edges that
+are incident to a triangle on the right and a triangle on the left are
+(topologically) flippable. These two triangles form then a
+quadrilateral, and the edge is one diagonal of this
+quadrilateral. Flipping the edge means that the quadrilateral is now
+split only the other diagonal. Flipping twice the same edge restores
+the original mesh.
+
+The following methods are useful for flipping an edge:
+- SurfaceMesh::isFlippable indicates if a given edge is flippable.
+- SurfaceMesh::otherDiagonal returns, given a flippable edge, the two other vertices of the quadrilateral.
+- SurfaceMesh::flip performs the flip on a flippable edge. The flipped edge keeps the same index as the original edge. You may choose whether or not you update locally the face normals.
+
+@note Vertex normals are not updated after a flip. Furthermore,
+neighbor faces of faces are not recomputed. You may recompute face
```

Comment:
I don't understand what you mean here. The flip is should be purely local. Do you mean something related to the face normal again?

Reply:
A SurfaceMesh precomputes more incidence relations or geometric informations to speed-up some processing of the surface. This is an advantage in some cases, but here it is not. This is why it is detailed what is recomputed and what is not. Face normales can be recomputed from positions since it is quite cheap to do so.